Stack/Opc.Ua.Core/Opc.Ua.Core.csproj

```diff
@@ -65,6 +65,7 @@
     <Reference Include="System.IdentityModel.Selectors" />
     <Reference Include="System.Security" />
     <Reference Include="System.ServiceModel" />
+    <Reference Include="Microsoft.CSharp" />
```

Comment:
>Microsoft.CShar [](start = 24, length = 15)

as discussed, lets try to get this working without dynamic types...

Reply:
removed reference.